OHS-Hosting-Infrastructure/complaint-tracker
# Enable using HSES staging data in dev

## Description of change

* Add a single environment variable `CT_USE_REAL_API_DATA` to turn on real API access in development
* Add `dotenv` gem to read configuration from `.env` file
* Add default values to `.env`
* Gem and NPM updates

## Acceptance Criteria

Given the app is running locally with `CT_USE_REAL_API_DATA=true`
When I visit the complaints list
Then I see complaints from HSES's staging environment

## How to test

1. Pull down code
2. Either set `CT_USE_REAL_API_DATA` to true in `.env` or create a new `.env.local` file with the content: `CT_USE_REAL_API_DATA=true`
3. Run `rails server`
4. Visit http://localhost:3000/complaints and verify HSES staging data is listed.

## Issue(s)

* closes #146 


## Checklist

To be completed by the submitter:



## To the Reviewer

This project is using [Conventional Comments](https://conventionalcomments.org/) to give structure
and context to PR comments. Please use these labels in your comments.

* **praise:**
* **nitpick:**
* **suggestion:**
* **issue:**
* **question:**
* **thought:**
* **chore:**
